games-board/cockatrice/cockatrice-20210126.ebuild

```diff
@@ -3,14 +3,15 @@
 
 EAPI=7
 
-inherit cmake
+inherit cmake xdg
```

Comment:
Thanks, merged, but a note about this.

It's good to have sorted inherits but (as mentioned before) have to be careful when multiple inherits export the same function, Soap had them reversed in the suggestion for a reason.

Unlike xdg-utils, xdg exports src_prepare, and so does cmake. xdg's src_prepare isn't useful here (it's often skipped) but without cmake's this won't build, and then the last one inherited will be executed giving us xdg's only.
```
 * ERROR: games-board/cockatrice-20210126::gentoo failed (configure phase):
 *   FATAL: cmake_src_prepare has not been run
```
So, I amended it to `inherit xdg cmake`

In the event the ebuild defined its own src_prepare, this wouldn't have mattered. There's also other rare special cases like some java eclasses needing a specific order but generally not a worry.

Reply:
Ahh, thx for the fix and for the clarification.
Need to be careful with these next time.